ci/scripts/utils/pr_list_database.py

```diff
@@ -101,15 +102,43 @@ def display_db(db, display):
     """
     values = []
     if len(display) == 1:
-        rows = db.fetch_data('pr_list', ['pr', 'state', 'status', 'reset_id', 'cases'], f'pr = {display[0]}')
-    else:
+        rows = db.fetch_data('pr_list', ['pr', 'state', 'status', 'reset_id', 'cases'], f"pr = '{display[0]}'")
+    if len(display) == 2:
+        rows = db.fetch_data('pr_list', ['pr'], f"state = '{display[0]}' AND status = '{display[1]}'")
+    if len(display) == 0:
         rows = db.fetch_data('pr_list', ['pr', 'state', 'status', 'reset_id', 'cases'])
     for row in rows:
         values.append(' '.join(map(str, row)))
 
     return values
 
 
+def update_database(db: SQLiteDB) -> list:
+    """
+    Update the database from the GitHub PRs
+    - only PRs from host machine are added to the database
+    - if the PR is already in the database it its added to the kill list
```

Comment:
I think I know why, but this is a bit counter-intuitive.

Reply:
I'm not sure that's something I need to "fix".